Source/Csla.Blazor/ViewModel.cs

```diff
@@ -387,7 +387,24 @@ public IPropertyInfo GetPropertyInfo<P>(Expression<Func<P>> property)
 
       var keyName = property.GetKey();
       var identifier = Microsoft.AspNetCore.Components.Forms.FieldIdentifier.Create(property);
-      return GetPropertyInfo(keyName, identifier.Model, identifier.FieldName);
+      return GetPropertyInfo(keyName, identifier.Model, identifier.FieldName, " ");
```

Comment:
Shouldn't this use the default rather than hard-coding the separator? Same for other instances below.

Reply:
Yes, you are correct. i will do that.